modules/dbnd/src/dbnd/_core/run/describe_run.py

```diff
@@ -146,19 +146,30 @@ def run_banner(self, msg, color="white", show_run_info=False, show_tasks_info=Tr
 
         return b.getvalue()
 
+    @property
+    def duration(self):
+        run_duration = 0
+        for task_run in self.run.task_runs:
+            try:
+                run_duration += task_run.duration
```

Comment:
does task_run has duration?

Reply:
yes, I've added this property in this commit